src/main/java/Events.java

```diff
@@ -0,0 +1,13 @@
+public class Events extends Task{
+    protected String at;
+
+    public Events (String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at + ")";
```

Comment:
You could consider storing the E symbol (and for the other tasks) as a constant to make it easier to edit them in future